doc/sphinx/linux_install_guide/requirement.rst

```diff
@@ -153,7 +155,7 @@ After that you can enter the option --with-biosig in the configure script of `St
 Building documentation
 ======================
 
-The manual of `Stimfit <http://www.stimfit.org>`_ including the documentation is accessible on-line in http://www.stimfit.org/doc/sphix/. To have your local copy, you will need to install sphinx:
+The manual of `Stimfit <http://www.stimfit.org>`_ including the documentation is accessible on-line in http://www.stimfit.org/doc/sphix/. To have your local copy, you will need to install sphinx version 1.7 or older:
```

Comment:
Are you sure that it has to be "older"?

Reply:
I tested 1.4.1 and returned an error. Better add  version =>1.7

Reply:
> Better add version =>1.7

That's version 1.7 _or newer_.
Greater than or equal is denoted by `>=`, _not_ `=>`. Please make sure it's the right way around in your pull request.